### DISCUSSION TITLE
Copy archive to a new HDD

### DISCUSSION THREAD

**Discussion by blueray453** [borgbackup/borg#7610] [Q&A] [answered]
I have created a borg archive like the following:

```
export BORG_LOGGING_CONF="/home/ismail/.config/borg/logging-WDPurple6TB.conf";
export BORG_CACHE_DIR=/media/ismail/WDPurple6TB/Borg_Cache;
export BORG_UNKNOWN_UNENCRYPTED_REPO_ACCESS_IS_OK=yes;

borg create --pattern=!/media/ismail/8TBRaid0/lost+found --pattern=!/media/ismail/8TBRaid0/_QEMU --pattern=!/media/ismail/8TBRaid0/_Backup/RSYNC --stats --progress --show-rc --show-version /media/ismail/WDPurple8TB/8TBRaid0_Backup::{hostname}-{now:%Y-%m-%d_%H:%M:%S} /media/ismail/8TBRaid0;
```
If I copy the contents of `/media/ismail/WDPurple8TB` to a new HDD then will it be considered a new archive.

I am planning to purchase a new hdd to make the same copy. And afterwards, make simultaneous backup on both hdd. I think copying will be faster if I am allowed to, because just running `borg create` will also go through the compression as well.

**Comment by infectormp** [comment, marked as answer]
I suggest you read this article [Can I copy or synchronize my repo to another location?](https://borgbackup.readthedocs.io/en/stable/faq.html#can-i-copy-or-synchronize-my-repo-to-another-location)
